distributor/example.html

```diff
@@ -1,7 +1,8 @@
-<body data-instant-allow-query-string data-instant-allow-external-links>
+<!-- See NOTEs below for changes you might want to make when productionizing. -->
+<body data-instant-allow-query-string>
 <p><a href="https://signed-exchange-testing.dev/sxgs/valid.html">HTML</a>
-<!-- TODO: Figure out why the image is not prefetching here. -->
 <p><a href="https://signed-exchange-testing.dev/sxgs/valid-image-subresource.html">HTML + image</a>
+<p><a href="https://twifkak.com/angular1/">HTML + lots of stuff</a>
```

Comment:
Consider moving the Angular example to `signed-exchange-testing.dev`.

Reply:
Yeah, that is on my mind, for future maintenance. (Although, in this case, having it on twifkak.com was helpful -- it surfaced the user-agent issue that I documented on distributor/README.md -- whereas s-e-t doesn't run CF ASX.)